R/table.R

```diff
@@ -42,3 +43,108 @@
     }
   )
 }
+
+# Converts data downloaded from Synapse to an appropriate data type in R
+.convertToRType <- function(list, synapseType) {
```

Comment:
create test(s) for converting back and forth and assuring the converted type is the same as the original